test/README.md

```diff
@@ -70,7 +70,7 @@ These tests require:
     ```
 3. A docker repo containing [the test images](#test-images)
 
-### Flags
+### Common Flags
```

Comment:
I see Srinivas has a few comments on the other PR (adding doc to pkg repo), once that's resolved, and we have reference to pkg test doc, I will approve.